jorge_flores/Calculator/src/Calculator/Calculator.java

```diff
@@ -0,0 +1,99 @@
+package Calculator;

```

Comment:
Package name 'Calculator' should be written in lower case to avoid conflict with the names of classes or interfaces